source/ncengine/graphics2/NcGraphicsImpl2.cpp

```diff
@@ -249,15 +254,15 @@ void NcGraphicsImpl2::Run()
 {
     NC_PROFILE_TASK("Render", Optick::Category::Rendering);
 
-    auto renderState = m_frontend.BuildRenderState(m_world);
-
     auto& context = m_engine.GetContext();
```

Comment:
Unrelated to this work, but so I don't forget: since this is our frame loop do we get any benefit from caching the pointers to context, device and swapchain here?

Reply:
No, really just context is referenced a bunch of times.